app-info/eos-extra.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<components version="0.8" origin="eos-extra-app-info">
+<components version="0.8" origin="flathub">
```

Comment:
I don't think I saw the reasoning for this in the commit message. Why does this need to be flathub?

Reply:
Yes, I missed to address this one. Appstream plugin sets the `source` for[ flatpak GsApp specifically](https://github.com/endlessm/gnome-software/blob/master/plugins/core/gs-appstream.c#L851-L859). Now, in general, flatpak repo's  appstream data contains the `origin` as `flatpak` (why is that, I don't know), but it's the flatpak plugin in gnome-software that [overrides the origin](https://github.com/endlessm/gnome-software/blob/master/plugins/flatpak/gs-flatpak.c#L575-L580) to be the specific(like `flathub`, `eos-apps` etc.)  flatpak remote.

Hence, now this has become the similar case of `<id>` that  I have mentioned in the commit message. As these apps that are being marked as popular, are coming from flathub remote, we need to set `flathub` as origin (otherwise while the GsApp with random `source`  is being adopted, the flatpak plugin  starts to warn/returns in the `_refine`, hence none of these apps are displayed in the `Featured` tab).  This is my understanding why we need this specific change.

Reply:
It's not correct to say that all of these apps come from Flathub. `com.google.Chrome` does not, for example.